openstf/pipeline/create_component_forecast.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+import structlog
+
+import pandas as pd
+
+import openstf.postprocessing.postprocessing as postprocessing
+
+
+def create_components_forecast_pipeline(pj, input_data, weather_data, split_coefs):
+    """Pipeline for creating a component forecast
+
+    Args:
+        pj (dict): Prediction job
+        input_data (pd.DataFrame): Input data fore the components forecast, this usually is a regular forecast
```

Comment:
Would it not be more clear to call it forecast in this case. The docstring does not explain what columns are expected here. I guess you need a very specific forecast dataframe to create the components forecast right?